### DISCUSSION TITLE
Handling replicates

### DISCUSSION THREAD

**Discussion by rtyags** [satijalab/seurat#4292] [Q&A]
Hi all,

What's the best way to handle replicates of two different conditions? If I have C1,C2 control replicates and T1,T2 treatment replicates, is it advisable to just integrate all 4 samples together? Doing that probably means we aren't using the power of having replicates. The other possibility might be to integrate replicates together (or perhaps just merge them?) and then integrate them across conditions? although given that it is usually not recommended to use the integration-transformed data for anything other than clustering, I'm not sure integrating already integrated data will be appropriate. So, maybe just merge C1,C2 and T1,T2 and then integrate them?

**Comment by timoast** [comment]
It depends what your goal is, but if you want to find differentially expressed genes between conditions, you could integrate all the datasets together to identify the common cell states, and then perform DE across conditions with replicate as a latent variable (using the uncorrected data)

**Reply by susheelbhanu** [reply]
Hi @timoast 
Is there a particular vignette that provides an example for this?

Thank you!

**Reply by AuritaP** [reply]
I am quite interested in this too
